doc/tutorial.rst

```diff
@@ -22,7 +25,52 @@ also the :doc:`primer` which focuses on a particular problem in detail.
 Users not familiar with the finite element method should start with the
 :ref:`sec-solving-pdes-fem`.
 
-Basic notions
+.. _invoking_from_command_line:
+
+Invoking *SfePy* from the Command Line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section introduces the basics of running *SfePy* from the command line.
+The ``$`` indicates the command prompt of your terminal.
+
+* The script `simple.py` is the most basic starting point in *SfePy*. It
+  can be invoked either directly::
+
+    $ ./simple.py <problem_description_file.py>
+
+  resp. ::
+
+    $ python ./simple.py <problem_description_file.py>
+
+  or use can use a main :ref:`command wrapper <SfePy-command-wrapper>`::
+
+    $ ./sfepy-run simple <problem_description_file.py>
+
+  resp. ::
+
+    $ sfepy-run simple <problem_description_file.py>
+
+  or even ::
+
+    $ python sfepy-run simple <problem_description_file.py>
+
```

Comment:
The above highly depends on:
1. is sfepy installed or being used in-place?
2. are we in its top-level directory or elsewhere?
3. is the current working directory on the binary search PATH?

As such, it IMHO needs more clarification.